src/Components/AsideMenu/Accordion/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './index.scss';
+class Accordion extends Component {
```

Comment:
```suggestion
import './index.scss';

class Accordion extends Component {
```
구분되는 요소들은 한줄씩 띄워주세요. 협업에 있어 가독성은 아주 중요한 요소입니다.

Reply:
수정했습니다.